site/es/blog/new-in-devtools-95/index.md

```diff
@@ -103,54 +124,65 @@ Chromium issues: [1076820](https://crbug.com/1076820), [1119900](https://crbug.c
 
+Ahora puedes hacer clic para leer las notas de la versión de DevTools en 6 otros idiomas - [Ruso](/ru/blog/new-in-devtools-95), [Chino](/zh/blog/new-in-devtools-95), [Español](/es/blog/new-in-devtools-95), [Japonés](/ja/blog/new-in-devtools-95), [Portugués](/pt/blog/new-in-devtools-95) y [Coreano](/ko/blog/new-in-devtools-95) a través de la pestaña *¿Qué hay de nuevo?*.
+
 <!-- Since Chrome 94, you can [set your preferred language](/blog/new-in-devtools-94/#localized) in DevTools. If you found any issues with the translations, help us improve it by [reporting a translation issue](https://goo.gle/devtools-translate) via **More options** > **Help** > **Report a translation bug**.  -->
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Qrg4Ahf4sYseL2NQZwIl.png", alt="New links to translated release notes and report a translation bug", width="800", height="487" %}
+Desde Chrome 94, puedes [establecer tu idioma preferido](/blog/new-in-devtools-94/#localized) en DevTools. Si encuentras algún problema con las traducciones, ayúdanos a mejorarlas [reportando un problema de traducción](https://goo.gle/devtools-translate) a través de **Más opciones** > **Ayuda** > **Reportar un problema de traducción**.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Qrg4Ahf4sYseL2NQZwIl.png", alt="Nuevos enlaces a las notas de la versión traducida y la posibilidad de reportar un error de traducción", width="800", height="487" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/312e43a6c50bc29f279f9eac2f91b723b36c7ee9 #}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/dcd3ae13ebc5d340b2abb07e9dc99cfa74caea35 #}
 
-Chromium issues: [1246245](https://crbug.com/1246245), [1245481](https://crbug.com/1245481) 
-
+Chromium issues: [1246245](https://crbug.com/1246245), [1245481](https://crbug.com/1245481)
 
 <!-- ## Improved UI for DevTools command menu {: #command-menu } -->
+## Mejorado el menú de comandos de DevTools {: #command-menu }
 
 <!-- Did you find it hard to search for a file in the [Command Menu](/docs/devtools/command-menu/#open)? Good news for you, the **Command Menu** user interface is now enhanced!  -->
+Has encontrado que es difícil buscar un archivo en el [Menú de comandos](/docs/devtools/command-menu/#open)? ¡Buenas noticias para ti, la interfaz de usuario del **Menú de comandos** se ha mejorado!
 
 <!-- Open the **Command Menu** to search for a file with keyboard shortcut <kbd>Control</kbd>+<kbd>P</kbd> in Windows and Linux, or <kbd>Command</kbd>+<kbd>P</kbd> in MacOS. -->
+Abre el **Menú de comandos** para buscar un archivo con atajo de teclado <kbd>Control</kbd>+<kbd>P</kbd> en Windows y Linux, o <kbd>Comando</kbd>+<kbd>P</kbd> en macOS.
```

Comment:
I think we don't need to translate the name of the 'Command' key as it has the same name in Spanish

Reply:
I have mixed feelings about this as I've seen Apple translating the Command key: https://support.apple.com/es-lamr/HT201255 But then you could find some official documents talking about Command. 😅